src/commands/deployContract.ts

```diff
@@ -68,48 +67,41 @@ async function register() {
             return;
         }
 
-        const buffer = new Serialize.SerialBuffer({
-            textEncoder: api.textEncoder,
-            textDecoder: api.textDecoder,
+        let abiJson = JSON.parse(fs.readFileSync(abiPath, 'utf-8'));
+
+        const encodedAbi = Serializer.encode({
+            object: abiJson,
         });
 
-        const abiDefinition = api.abiTypes.get(`abi_def`);
-        let abi = JSON.parse(fs.readFileSync(abiPath, 'utf-8'));
-        abi = abiDefinition.fields.reduce(
-            (acc, { name: fieldName }) => Object.assign(acc, { [fieldName]: acc[fieldName] || [] }),
-            abi
-        );
-        abiDefinition.serialize(buffer, abi);
+        const abiHex = encodedAbi.toString('hex');
```

Comment:
This does not seem to work when deploying a contract. Results in an error:
`Unable to determine the type of object to be encoded. To encode custom ABI types you must pass the type argument`

Maybe should be like in the `ultra.proposer`:
```
let abi = JSON.parse(fs.readFileSync(abiPath, `utf8`))
const encoded = Serializer.encode({object: ABI.from(abi)})
abiHex = Buffer.from(encoded.array).toString(`hex`)
```